yarn/src/main/scala/org/apache/spark/deploy/yarn/ClientArguments.scala

```diff
@@ -73,12 +73,12 @@ private[spark] class ClientArguments(args: Array[String], sparkConf: SparkConf)
       .orNull
     // If dynamic allocation is enabled, start at the max number of executors
     if (isDynamicAllocationEnabled) {
-      val maxExecutorsConf = "spark.dynamicAllocation.maxExecutors"
-      if (!sparkConf.contains(maxExecutorsConf)) {
+      val minExecutorsConf = "spark.dynamicAllocation.maxExecutors"
+      if (!sparkConf.contains(minExecutorsConf)) {
```

Comment:
I think the rationale may have been that the `ExecutorAllocationManager` and client can run in different processes.